### PR TITLE
Tendermint proxy for clients

### LINKED ISSUE #431: Proxy client requests to Tendermint RPC
Currently client is required to calculate Tendermint port to interact with the cluster, and then to connect to each and every port independently.

Proposition: make a routing layer in the Node, so that client may access the cluster's Tendermint RPC via the single port.

### PR DESCRIPTION
**Pull request by alari**
Fixes #431

### CODE REVIEW

File: node/src/main/scala/fluence/node/status/StatusHttp.scala [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    HttpRoutes
+      .of[F] {
+        case GET -> Root =>
+          for {
+            status <- sm.getStatus
+            json <- Sync[F].delay(status.asJson.spaces2).handleError {
+              case NonFatal(e) =>
+                e.printStackTrace()
+                logger.error(s"Status cannot be serialized to JSON. Status: $status", e)
+                "\"JSON generation errored, please try again\""
```
Comment (folex) [inline node/src/main/scala/fluence/node/status/StatusHttp.scala]: `HTTP 200: JSON generation errored, please try again` is a bad idea

Comment (folex) [reply node/src/main/scala/fluence/node/status/StatusHttp.scala]: also, please use `"""` so avoid escaping

File: node/src/it/scala/fluence.node/MasterNodeIntegrationSpec.scala [hunk 2]
```diff
@@ -49,6 +49,9 @@ class MasterNodeIntegrationSpec
 
   private val sttpResource = Resource.make(IO(AsyncHttpClientCatsBackend[IO]()))(sttpBackend ⇒ IO(sttpBackend.close()))
 
+  // TODO integrate with CLI, get app id from tx
```
[the remark this answers is not in the document]
Comment (alari) [reply node/src/it/scala/fluence.node/MasterNodeIntegrationSpec.scala]: - It's an integration test that takes almost all the moving parts into account, not just Scala
- There's no reason to implement complex logic, like getting app id from tx, in Scala. But we need it for proper testing

File: node/src/main/resources/reference.conf [hunk 3]
```diff
@@ -34,8 +34,8 @@ swarm {
   // host = "http://localhost:8500"
 }
 
-// the server that provides statistics about a master node and workers
-status-server {
+// Main HTTP API server config
```
[the remark this answers is not in the document]
Comment (alari) [reply node/src/main/resources/reference.conf]: Master node is going to have the only HTTP API endpoint, I suppose? Now it contains only two services, but later could have some more. Why to explain it in configuration file?

**Pull request merged**